sticky_notes.py

```diff
@@ -21,17 +21,19 @@ def stickynote(self, event):
         if user not in self.notes:
             self.notes[user] = []
 
-        self.notes[user].append("Note from {0}: {1}".format(user, content))
+        notes = self.notes[user]
+        notes.append("Note from {0}: {1}".format(event.meta['nick'], content))
+        self.notes[user] = notes
```

Comment:
@pepol calling `self.notes[user].append('stuff')` won't do much, especially not what you'd think it would.